star-rx72n-firmware/lib/rx_hal/src/rx_gptw.c

```
@@ -687,35 +687,189 @@ rx_err_t rx_gptw_stop(const rx_gptw_channel_t channel)
+
+/**
+ * @brief Calculate initial counter value and direction for phase staggering
+ *
+ * @param[in] channel GPTW channel
+ * @param[in] period PWM period in counts
+ * @param[in] is_triangle True if triangle mode (center-aligned)
+ * @param[out] init_count Pointer to store initial counter value
+ * @param[out] init_dir_up Pointer to store initial direction (true=up, false=down)
+ *                         Only valid for triangle mode.
+ */
+static void internal_calculate_phase_offset(const rx_gptw_channel_t channel,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'internal_calculate_phase_offset' of similar type ('const int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)